bridge/configuration.php

```diff
@@ -114,7 +114,7 @@ function _ddtrace_config_indexed_array($value, $default)
 
     return array_map(
         function ($entry) {
-            return strtolower(trim($entry));
+            return trim($entry);
```

Comment:
This is generic getter and should not make assumption about case sensitive vs case insensitive. Also, this function was not used by other settings so it is good to be changed as it is only used by the newly added functions.